examples/all-clusters-app/esp32/CMakeLists.txt

```diff
@@ -29,8 +29,6 @@ if(${IDF_TARGET} STREQUAL "esp32")
     list(APPEND EXTRA_COMPONENT_DIRS "${CMAKE_CURRENT_LIST_DIR}/../../common/m5stack-tft/repo/components/tft"
                                      "${CMAKE_CURRENT_LIST_DIR}/../../common/m5stack-tft/repo/components/spidriver"
                                      "${CMAKE_CURRENT_LIST_DIR}/../../common/screen-framework")
-elseif(${IDF_TARGET} STREQUAL "esp32c3")
-    list(APPEND EXTRA_COMPONENT_DIRS "$ENV{IDF_PATH}/examples/peripherals/rmt/led_strip/components")
```

Comment:
@zwx1995esp Why is this removed? The ESP32 C3 code doesn't use this?

With this change, how is the led_strip components/code included if this is used by C3?

Reply:
Hi,  dhrishi , with the tag `release/v4.4` of the `ESP-IDF`, there is a commit (cccdb3e) which has already moved `examples/peripherals/rmt/led_strip/component/led_strip` to the `examples/common_components`. In the commit `05cdfbfa3c` we have already added  the new dirs `${IDF_PATH}/examples/common_components`, but did not removed this unused external component.